.github/workflows/tools-releases.yml

```diff
@@ -55,25 +41,18 @@ jobs:
               ETCD_VERSION=$(yq eval '.etcd' $changed_file)
               echo "ETCD_VERSION=$ETCD_VERSION" >> $GITHUB_ENV
             done
-      - name: Set up Docker
-        uses: docker/setup-buildx-action@d70bba72b1f3fd22344832f00baa16ece964efeb # tag=v3.3.0
-      - name: Log in to the Container registry
-        uses: docker/login-action@e92390c5fb421da1463c202d546fed0ec5c39f20 # tag=v3.1.0
-        with:
-          registry: ghcr.io
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Build and push Docker images
-        uses: docker/build-push-action@2cdde995de11925a030ce8070c3d77a52ffcf1c0 # tag=v5.3.0
+      - name: Build packages
+        run: |
+          make release-envtest \
+            KUBERNETES_VERSION=${{ env.KUBERNETES_VERSION }} \
+            GO_VERSION=${{ env.GO_VERSION }} \
+            ETCD_VERSION=${{ env.ETCD_VERSION }}
+      - name: Release
+        uses: softprops/action-gh-release@9d7c94cfd0a1f3ed45544c887983e9fa900f0564 # tag=v2.0.4
         with:
-          context: .
-          file: ./hack/envtest/${{matrix.os}}/Dockerfile
-          build-args: |
-            GO_VERSION=${{env.GO_VERSION}}
-            KUBERNETES_VERSION=${{env.KUBERNETES_VERSION}}
-            ETCD_VERSION=${{env.ETCD_VERSION}}
-            OS=${{matrix.os}}
-            ARCH=${{matrix.arch}}
-          push: true
-          tags: |
-            ghcr.io/kubernetes-sigs/controller-tools/envtest:${{env.KUBERNETES_VERSION}}-${{matrix.os}}-${{matrix.arch}}
+          name: envtest-${{ env.KUBERNETES_VERSION }}
```

Comment:
Should we use envtest/${{  env.KUBERNETES_VERSION }}

Then it would be consistent with tagging for go submodules